tappsbt/interface.go

```diff
@@ -173,6 +188,23 @@ func (p *VPacket) Copy() *VPacket {
 	}
 }
 
+// CommitmentVersion returns the Taproot Asset commitment version that matches
+// the VPacket version.
+func CommitmentVersion(vers VPacketVersion) (*commitment.TapCommitmentVersion,
```

Comment:
Doesn't seem to be used?

Reply:
Used in `tapsend/commitPacket`, `tapsend/ValidateAnchorOutputs`, `tapfreigher/wallet/FundPacket`. And the aux funding controller.